### DISCUSSION TITLE
how to understand the input's images resolution before start training?

### DISCUSSION THREAD

**Discussion by t00350320** [kohya-ss/sd-scripts#1027] [General]
generally speaking， sd supports 512,512 image size training, so the question is, if i crop a face image in random size like 1444*1561, can i throw this image into ss scripts immediately without transfering to 512*512 size?

**Comment by DKnight54** [comment]
If you enable bucketing and did not disable resizing, in theory yes.
The script attempts to put images into as close together buckets as possible, and resize and crop the images to fit. However, some people have feedbacked that they get better results by manually fitting images into the buckets to ensure as even a distribution as possible.
